test/cluster/test_tablets2.py

```
@@ -859,6 +859,79 @@ async def test_concurrent_table_drop_and_major(manager: ManagerClient):
+        await manager.api.flush_keyspace(src.ip_addr, ks)
+
+        dst = await manager.server_add()
+        dst_host_id = await manager.get_host_id(dst.server_id)
+
+        src_log = await manager.server_open_log(src.server_id)
+        src_mark = await src_log.mark()
+
+        await manager.api.enable_injection(src.ip_addr, "major_compaction_wait", one_shot=False)
+
+        logger.info("Starting table-wide major compaction")
+        compaction = asyncio.create_task(manager.api.keyspace_compaction(src.ip_addr, ks))
```

Ruff (0.16.1)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)